.eslintrc.js

```diff
@@ -6,20 +6,44 @@ module.exports = {
     'no-magic-numbers': 'off',
     'new-cap': 'off',
     'comma-dangle': 'off',
-    'import/order': ['error', { 'newlines-between': 'always' }]
+    'import/order': [
+      'error',
+      {
+        'newlines-between': 'always',
+        alphabetize: { order: 'asc', caseInsensitive: true },
+        pathGroups: [
+          {
+            pattern: 'react',
+            group: 'external',
+            position: 'before'
+          },
+          {
+            pattern: 'react-native',
+            group: 'external',
+            position: 'before'
+          }
+        ],
+        pathGroupsExcludedImportTypes: ['react', 'react-native']
+      }
+    ]
   },
   settings: {
     'import/ignore': ['node_modules'],
     'import/resolver': {
-      node: {
-        paths: ['src'],
-        settings: {
-          'import/resolver': {
-            node: {
-              paths: ['src'],
-              extensions: ['.ios.js', '.android.js', '.js', '.jsx', '.ts', '.tsx', '.json']
-            }
-          }
+      'babel-module': {
+        root: ['./src'],
+        extensions: ['.js', '.jsx', '.ts', '.tsx'],
+        alias: {
+          '@app': './src/app',
+          '@assets': './src/app/assets',
+          '@components': './src/app/components',
+          '@screens': './src/app/screens',
+          '@config': './src/config',
+          '@constants': './src/constants',
+          '@interfaces': './src/interfaces',
+          '@redux': './src/redux',
+          '@services': './src/services',
+          '@utils': './src/utils'
```

Comment:
Mmm why this change? The aliases are in `babel.config.js` and `tsconfig.json`, not here 🤔

Reply:
Sorry. I'm still understanding this. But, you have reason.